docs/src/understand/thin-edge-json.md

```diff
@@ -1,52 +1,107 @@
 ---
 title: Thin Edge JSON
 tags: [Concept, MQTT]
-sidebar_position: 1
+sidebar_position: 4
 ---
 
 # Thin Edge JSON format
 
-Thin Edge JSON is a lightweight format used in `thin-edge.io` to represent measurements data.
-This format can be used to represent single-valued measurements, multi-valued measurements
+Thin Edge JSON is a lightweight format used in `thin-edge.io` to represent telemetry data: measurements, events and alarms.
+These data are exchanged over the [MQTT bus](mqtt-bus.md) and sent by the devices and services of a agent.
+
+## Sending Telemetry Data
+
+The `thin-edge.io` framework exposes some MQTT endpoints that can be used by local processes
+to exchange data between themselves as well as to get some data forwarded to the cloud.
+It will essentially act like an MQTT broker against which you can write your application logic.
```

Comment:
It is an MQTT broker, so it is not "acting" as one.

I'm not 100% sure what the purpose of the sentence is, is it to indicate the the MQTT broker should be the main source of data which is used within application logic?

Reply:
Fully agree, that's confusing. I have to rewrite this paragraph.

Reply:
For now simply removed